src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'categories/CHECK_STATUS';
```

Comment:
- [ ] The value of  action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       const CHECK_STATUS = bookstore-ivanmvh/categories/CHECK_STATUS